taskcluster/app_services_taskgraph/transforms/toolchain.py

```diff
@@ -41,3 +41,10 @@ def resolve_keys(config, tasks):
             routes.append(TOOLCHAIN_OLD_INDEX[task['name']].format(sha=sha))
 
         yield task
+
+@transforms.add
+def fixup_mac_worker(config, tasks):
```

Comment:
nit: technically `mac` isn't the commonality here, it's `generic-worker`

But couldn't we just avoid adding `docker-image` for generic-worker tasks in the first place (like you do in the `setup_worker` transforms below)?

Actually looks like the `toolchain` kind is already using those transforms, so I'm not sure this is even necessary?

Reply:
I'm pretty sure you're right it wasn't necessary anymore after I added the worker transform.  I just deleted that code and am doing one last `[full ci]` check to make sure.